woocommerce/woocommerce-ios
# [CPP Onboarding] Implement Stripe account under review

Part of #4611 

This adds an specific error when the Stripe account is still under review.

The error screen:
<img src="https://user-images.githubusercontent.com/2722505/128885953-c382ac4c-eaf6-4f0c-8a39-b791151a8f97.png" width="350"/>

When tapping Contact us, we should redirect the user to contact support:
<img src="https://user-images.githubusercontent.com/2722505/128886137-9f2ac0aa-8fd6-45ed-af79-7c0340029a73.png" width="350"/>

Learn more should open a web view:
<img src="https://user-images.githubusercontent.com/2722505/128886273-d0cbb92f-549b-406f-a66b-d8c355298666.png" width="350"/>


## How to test
In order to test this in real conditions, it is necessary to have a store associated to an Stripe account that is under review, or...
make `CardPresentPaymentsOnboardingUseCase.checkOnboardingState` return `stripeAccountUnderReview`

1. Navigate to Settings > In-Person Payments
2. Notice the error screen.


Update release notes:
